api/v1/countries.py

```diff
@@ -21,7 +21,8 @@ async def _count_aed_in_country(country: Country, aed_state: AEDState, send_stre
 
 @router.get('/names')
 @configure_cache(timedelta(hours=1), stale=timedelta(days=7))
-async def get_names(request: Request, country_state: CountryStateDep, aed_state: AEDStateDep):
+async def get_names(request: Request, country_state: CountryStateDep, aed_state: AEDStateDep, language: str | None = None):
+    target_language = language.upper() if language is not None else None
```

Comment:
I don't think using upper() here is a good idea. The client should be responsible for case handling, and doing this on our end prevents language filtering by "default," which is lowercase.
```suggestion
```